redis/production.cfg

```diff
@@ -0,0 +1,17 @@
+# collective.taskqueue configuration, setting up a redis as persistent queue.
+# Requires:
+# - production.cfg
+# - redis/[version].cfg
+
+[buildout]
+extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/redis/base.cfg
+parts += ${redis-settings:parts}
+
+
+[redis-settings]
+port = 1${buildout:deployment-number}33
```

Comment:
Sure, done.
See also https://github.com/4teamwork/ftw-buildouts/blob/master/README.rst#port-range-configuration , which is the newer version of this table 😉